src/app/component/navbar/navbar.component.html

```diff
@@ -32,55 +32,72 @@
             <select [(ngModel)]="selectedCountries" (ngModelChange)="countriesSelected(selectedCountries)">
               <option value="" disabled selected hidden>pays</option>
               <option *ngFor="let country of countries" [value]="country.countryName">{{ country.countryName }}</option>
-            </select>0
+            </select>
+            0
             <select place [(ngModel)]="selectedAllergens" (ngModelChange)="allergensSelected(selectedAllergens)">
               <option value="" disabled selected hidden>allergens</option>
               <option *ngFor="let allergen of allergens" [value]="allergen.allergenName">
                 {{ allergen.allergenName }}
               </option>
             </select>
             <select [(ngModel)]="selectedDiets" (ngModelChange)="dietsSelected(selectedDiets)">
-              <option value="" disabled selected hidden> diets</option>
+              <option value="" disabled selected hidden>diets</option>
               <option *ngFor="let diet of diets" [ngValue]="diet.dietName">{{ diet.dietName }}</option>
             </select>
           </div>
-          <div class="search_burger--unselected" >
+          <div class="search_burger--unselected">
             <button (click)="countriesSelected(countries)" *ngFor="let countries of countriesList">
-              {{ countries }} <b>x</b>
+              {{ countries }}
+              <b>x</b>
             </button>
             <button (click)="allergensSelected(allergens)" *ngFor="let allergens of allergensList">
-              {{ allergens }} <b>x</b>
+              {{ allergens }}
+              <b>x</b>
+            </button>
+            <button (click)="dietsSelected(diets)" *ngFor="let diets of dietsList">
+              {{ diets }}
+              <b>x</b>
             </button>
-            <button (click)="dietsSelected(diets)" *ngFor="let diets of dietsList">{{ diets }} <b>x</b></button>
           </div>
-      </div>
+        </div>
       </div>
     </div>
   </div>
 
   <nav class="links_menu">
     <ul>
+      <div *ngIf="isLoggedIn(); else loginButton">
+        <button (click)="logout()">Logout</button>
+      </div>
+      <ng-template #loginButton>
+        <button (click)="showLoginModal()">Login</button>
+      </ng-template>
+
+      <button (click)="openLoginModal()">
+        <lucide-icon name="chef-hat" class="login-icon"></lucide-icon>
+        <p>Login</p>
+      </button>
       <li>
         <a>
-          <lucide-icon class="links_menu-icon" name="chef-hat" size="35" strokeWidth="2.5"></lucide-icon>
+          <lucide-icon class="links_menu-icon" name="chef-hat"></lucide-icon>
           <p>Community</p>
         </a>
       </li>
       <li>
         <a routerLink="favourites">
-          <lucide-icon class="links_menu-icon" name="star" size="35" strokeWidth="2.5"></lucide-icon>
+          <lucide-icon class="links_menu-icon" name="star"></lucide-icon>
           <p>Favourites</p>
```

Comment:
avoid p tag since it is more a title than a paragraph, use a coherent h tag to fit with the othe h of the page